packages/number-field/src/NumberField.ts

```diff
@@ -184,6 +184,13 @@ export class NumberField extends TextfieldBase {
     private change!: (event: PointerEvent) => void;
     private safty!: number;
 
+    constructor() {
+        super();
+        new LanguageResolutionControler(this, (language: string) => {
+            this.resolvedLanguage = language;
+        });
```

Comment:
Not a criticism of this PR, but creating "anonymous" controllers that reference the host, that the host has no reference to, feels weird to me. I'm not sure how I feel about that part of the ReactiveController API.

Reply:
I think the question is, do we think a callback API or a property API is better to consume here. I'm looking at taking `language` off of the API because it's not currently "reactive", but maybe we should lean more into that workflow for consuming _this_ Reactive Controller?

Reply:
That's a good point - I don't think I have a strong opinion about which direction the API guides users in, but ideally it would be one obvious way.

I have a _slight_ preference towards making `this.langController.language` reactive and getting rid of the callback, because:

- that seems to align with the emerging field of reactive controllers generally
- it's harder for a user to fail / puts less on the user (they don't need to understand what to put into the callback & why)

But if you'd prefer dropping `language` I'd be fine with that too, at least then the callback requirement would be clear from the shape of the API.

Reply:
Very helpful! I've updated to a reactive `this.languageResolver.language` value for this API. Stones and birds. 😉